tests/test_http_parser.py

```diff
@@ -826,6 +826,58 @@ class TestParsePayload:
         assert isinstance(out.exception(),
                           http_exceptions.TransferEncodingError)
 
+    @pytest.mark.xfail(
+        raises=AssertionError,
+        reason="see https://github.com/aio-libs/aiohttp/issues/4630"
+    )
```

Comment:
You don't need to specify `raises=AssertionError`. It's usually needed to specify other exceptions that happen, not assertions.
```suggestion
    @pytest.mark.xfail(reason="see https://github.com/aio-libs/aiohttp/issues/4630")
```